libs/components/pages/src/lib/modules/needs-attention/needs-attention.component.scss

```diff
@@ -9,30 +9,27 @@
   display: flex;
   flex-wrap: nowrap;
   align-items: center;
+  cursor: pointer;
```

Comment:
Is `cursor: pointer` necessary? Both the default style for `a` and `.sky-btn` already specify this.

Reply:
This ended up being an issue where `skyHref` no longer allowed `#` as a link, so the link was styled as non-active.